[BUG] Transaction coordinator initialize failed when start broker with kop

**Describe the bug**
When turn on transaction for KOP and start pulsar broker with KOP, it throws `Initialized transaction coordinator failed` exception. The exception log list as follow.
```
10:56:47.910 [main] ERROR io.streamnative.pulsar.handlers.kop.KafkaProtocolHandler - Initialized transaction coordinator failed.
java.util.concurrent.ExecutionException: java.lang.Exception: Failed to create path /kop_latest_producer_id_block keeperException code -5
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395) ~[?:?]
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999) ~[?:?]
	at io.streamnative.pulsar.handlers.kop.KafkaProtocolHandler.startTransactionCoordinator(KafkaProtocolHandler.java:386) ~[?:?]
	at io.streamnative.pulsar.handlers.kop.KafkaProtocolHandler.start(KafkaProtocolHandler.java:271) ~[?:?]
	at org.apache.pulsar.broker.protocol.ProtocolHandlerWithClassLoader.start(ProtocolHandlerWithClassLoader.java:66) ~[org.apache.pulsar-pulsar-broker-2.8.0-rc-202101252233.jar:2.8.0-rc-202101252233]
	at org.apache.pulsar.broker.protocol.ProtocolHandlers.lambda$start$4(ProtocolHandlers.java:142) ~[org.apache.pulsar-pulsar-broker-2.8.0-rc-202101252233.jar:2.8.0-rc-202101252233]
	at java.lang.Iterable.forEach(Iterable.java:75) [?:?]
	at org.apache.pulsar.broker.protocol.ProtocolHandlers.start(ProtocolHandlers.java:142) [org.apache.pulsar-pulsar-broker-2.8.0-rc-202101252233.jar:2.8.0-rc-202101252233]
	at org.apache.pulsar.broker.PulsarService.start(PulsarService.java:647) [org.apache.pulsar-pulsar-broker-2.8.0-rc-202101252233.jar:2.8.0-rc-202101252233]
	at org.apache.pulsar.PulsarBrokerStarter$BrokerStarter.start(PulsarBrokerStarter.java:255) [org.apache.pulsar-pulsar-broker-2.8.0-rc-202101252233.jar:2.8.0-rc-202101252233]
	at org.apache.pulsar.PulsarBrokerStarter.main(PulsarBrokerStarter.java:326) [org.apache.pulsar-pulsar-broker-2.8.0-rc-202101252233.jar:2.8.0-rc-202101252233]
Caused by: java.lang.Exception: Failed to create path /kop_latest_producer_id_block keeperException code -5
	at io.streamnative.pulsar.handlers.kop.coordinator.transaction.ProducerIdManager.lambda$makeSurePathExists$2(ProducerIdManager.java:223) ~[?:?]
	at org.apache.bookkeeper.zookeeper.ZooKeeperClient$10$1.processResult(ZooKeeperClient.java:740) ~[org.apache.bookkeeper-bookkeeper-server-4.12.1.jar:4.12.1]
	at org.apache.zookeeper.ClientCnxn$EventThread.processEvent(ClientCnxn.java:642) ~[org.apache.pulsar-pulsar-zookeeper-2.8.0-rc-202101252233.jar:2.8.0-rc-202101252233]
	at org.apache.zookeeper.ClientCnxn$EventThread.run(ClientCnxn.java:510) ~[org.apache.pulsar-pulsar-zookeeper-2.8.0-rc-202101252233.jar:2.8.0-rc-202101252233]
```

**To Reproduce**
1. Set `enableTransactionCoordinator=true` in broker.conf
2. Use `bin/pulsar-daemon start broker` to start pulsar broker 
3. See error in broker log

--- a/kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/transaction/ProducerIdManager.java
+++ b/kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/transaction/ProducerIdManager.java
@@ -30,6 +30,7 @@ import lombok.extern.slf4j.Slf4j;
 import org.apache.zookeeper.AsyncCallback;
 import org.apache.zookeeper.CreateMode;
 import org.apache.zookeeper.KeeperException;
+import org.apache.zookeeper.ZooDefs;
 import org.apache.zookeeper.ZooKeeper;
 import org.apache.zookeeper.data.Stat;
 
@@ -218,14 +219,15 @@ public class ProducerIdManager {
 
     private CompletableFuture<Void> makeSurePathExists() {
         CompletableFuture<Void> completableFuture = new CompletableFuture<>();
-        zkClient.create(KOP_PID_BLOCK_ZNODE, null, null, CreateMode.PERSISTENT, (rc, path, ctx, name) -> {
-            if (rc != KeeperException.Code.OK.intValue() && rc != KeeperException.Code.NODEEXISTS.intValue()) {
-                completableFuture.completeExceptionally(
-                        new Exception("Failed to create path " + KOP_PID_BLOCK_ZNODE + " keeperException code " + rc));
-                return;
-            }
-            completableFuture.complete(null);
-        }, null);
+        zkClient.create(KOP_PID_BLOCK_ZNODE, null, ZooDefs.Ids.OPEN_ACL_UNSAFE,
+                CreateMode.PERSISTENT, (rc, path, ctx, name) -> {
+                    if (rc != KeeperException.Code.OK.intValue() && rc != KeeperException.Code.NODEEXISTS.intValue()) {
+                        completableFuture.completeExceptionally(
+                                new Exception("Failed to create path " + KOP_PID_BLOCK_ZNODE + " keeperException code " + rc));
+                        return;
+                    }
+                    completableFuture.complete(null);
+                }, null);
         return completableFuture;
     }
 

--- a/kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/transaction/ProducerIdManager.java
+++ b/kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/transaction/ProducerIdManager.java
@@ -223,7 +223,8 @@ public class ProducerIdManager {
                 CreateMode.PERSISTENT, (rc, path, ctx, name) -> {
                     if (rc != KeeperException.Code.OK.intValue() && rc != KeeperException.Code.NODEEXISTS.intValue()) {
                         completableFuture.completeExceptionally(
-                                new Exception("Failed to create path " + KOP_PID_BLOCK_ZNODE + " keeperException code " + rc));
+                                new Exception("Failed to create path " + KOP_PID_BLOCK_ZNODE
+                                        + " keeperException code " + rc));
                         return;
                     }
                     completableFuture.complete(null);

